src/test/java/com/google/api/codegen/testdata/discoveries/go/go_storage.v1.json.baseline

```diff
@@ -1474,6 +1474,8 @@ func main() {
     // TODO: Add desired fields of the request body.
   }
 
+  // TODO: Use `Download` in place of `Do` below to download media content. For more information, see:
+  // https://godoc.org/google.golang.org/api/storage/v1
```

Comment:
Since this is autogen doc, I would expect them to be deterministic and hence stable as long as the method does not disappear. Moreover, IIRC a non-existent anchor still loads the page, so we'd be no worse off.

@saicheems Thoughts on the link stability?